Refresh of the PWA docs (#2334)
MicrosoftDocs/edge-developer: microsoft-edge/progressive-web-apps-chromium/how-to/web-app-manifests.md

| `display_override` | Used to determine the preferred display for the app, depending on what the device supports. |
| `theme_color` | The default theme color for the app. This affects how the OS displays the site. |
| `background_color` | The background color of the window where the app is launched, before the stylesheet is applied. |
| `orientation` | On supporting devices, this defines the default orientation for the app (such as landscape or portrait). |
| `protocol_handlers` | The list of predefined or custom URI protocol schemes the app is associated with. |
| `shortcuts` | The list of common tasks users will be able to do by right-clicking or long-pressing on the app icon. |
| `share_target` | The type of data that the app accepts when content is shared via the OS share dialog. |
| `file_handlers` | The type of files that the app can be launched with. |

To learn more about manifest members, see [web app manifests](https://developer.mozilla.org/docs/Web/Manifest) at MDN.

<!-- ====================================================================== -->
## Handle specific protocol schemes

> You can register your PWA as a protocol handler. When a user clicks on a link that uses the protocol scheme your app registered to, your app is launched automatically. This can be used by users to easily share links to specific parts or functionality of your app with each other.

Reviewer: ```suggestion
You can register your PWA as a protocol handler. When a user clicks on a link that uses the protocol scheme your app registered to, your app is launched automatically. This enables users to easily share links to specific parts or functionality of your app with each other.
```
had trouble parsing the series of words that includes "used by users"
